Article : Panoramax, quel bilan tirer de la phase de construction ? (#989)
geotribu/website: content/articles/2023/2023-10-02_panoramax-bilan-et-nouvelle-etape.md

---
title: "Panoramax franchit une nouvelle étape"
subtitle: Photo-cartographions librement le monde
authors:
    - Mathilde FERREY
    - Christian QUEST
categories:
    - article
comments: true
date: "2023-10-03 10:20"
description: "C'était il y a un an... l'IGN répondait à la proposition d'OpenStreetMap France et se lançait dans une phase d'investigation pour la création d'un street-view libre. Aujourd'hui, où en est-on ?"

> image: "Image d'illustration de l'article qui sert ensuite dans la mise en avant : réseaux sociaux, flux RSS... 400x800 en PNG"

Reviewer: ```suggestion
icon: material/panorama-variant
image: "Image d'illustration de l'article qui sert ensuite dans la mise en avant : réseaux sociaux, flux RSS... 400x800 en PNG"
```

license: default
robots: index, follow
tags:

>     - Geovisio
>     - Panoramax
>     - vues immersives

Reviewer: ```suggestion
    - Geovisio
    - IGN
    - OpenStreetMap
    - Panoramax
    - vues immersives
```

---

# Panoramax, l'alternative libre pour photo-cartographier les territoires, franchit une nouvelle étape

:calendar: Date de publication initiale : 3 octobre 2023

> 
> C'était il y a un an... l'IGN répondait à la proposition d'OpenStreetMap France et se lançait dans une phase d'investigation pour la création d'un Street-View libre nommé Panoramax. Aujourd'hui, où en est-on ?

Reviewer 2: Une petite vignette avec le logo c'est tjs cympa non ?

```suggestion

![logo Panoramax](https://cdn.geotribu.fr/img/logos-icones/entreprises_association/panoramax.jpg "logo Panoramax"){: .img-rdp-news-thumb }

C'était il y a un an... l'IGN répondait à la proposition d'OpenStreetMap France et se lançait dans une phase d'investigation pour la création d'un Street-View libre nommé Panoramax. Aujourd'hui, où en est-on ?
```

> C'était il y a un an... l'IGN répondait à la proposition d'OpenStreetMap France et se lançait dans une phase d'investigation pour la création d'un Street-View libre nommé Panoramax. Aujourd'hui, où en est-on ?

Reviewer: ```suggestion
C'était il y a un an... l'[IGN](https://ign.fr) répondait à la proposition d'[OpenStreetMap France](https://www.openstreetmap.fr) et se lançait dans une phase d'investigation pour la création d'un Street-View libre nommé [Panoramax](https://panoramax.fr). Aujourd'hui, où en est-on ?
```

Le projet tient-il ses promesses et les contributeurs sont-ils bien au rendez-vous ?

[Commenter cet article :fontawesome-solid-comments:](#__comments){: .md-button }
{: align=middle }

----

## Quelques chiffres

Fin septembre 2023, Panoramax c'est :

> * 2 instances publiques (IGN + OSM-FR)

Reviewer 3: - https://panoramax.openstreetmap.fr/#focus=map&map=2.01/12.45/31.69&speed=250
- https://panoramax.ign.fr/#focus=map&map=5.7/47/3&speed=250

> * 2 instances publiques (IGN + OSM-FR)

Reviewer: ```suggestion
* 2 instances publiques ([IGN](https://panoramax.ign.fr) + [OSM-FR](https://panoramax.openstreetmap.fr))
```

* Plus de 8 millions de photos

> * Plus de 20.000 séquences

Reviewer: Est-ce qu'on a un tableau de bord public qui permet de suivre le versement en temps réel?

Author: pas encore

* Une cinquantaine de contributeurs

> * Un début d’intégration QGIS (avec Oslandia)

Reviewer: ```suggestion
* Un début d’intégration [QGIS](https://www.qgis.org/) (avec [Oslandia](https://oslandia.com))
```

----

## Que propose Panoramax aujourd'hui?

Panoramax, c'est avant tout un géo-commun construit sur des principes fondateurs :

* Partager ses photos pour qu'un même cliché géolocalisé soit utilisable par tous

> * Une multiplicité d’acteurs tant pour les prises de vues que pour les réutilisations

Reviewer 3: Mais on peut dire prise**s** de vue quand même vu qu'il y en a 8 millions ?

> * Une multiplicité d’acteurs tant pour les prises de vues que pour les réutilisations

Reviewer 4: Oui oui, la remarque poste sur le mot "vue"

> * Une multiplicité d’acteurs tant pour les prises de vues que pour les réutilisations

Author: bien vu ;-)

* Un projet décentralisé soutenu par une communauté élargie

![Panoramax - Projet décentralisé](https://cdn.geotribu.fr/img/articles-blog-rdp/articles/2023/panoramax/panoramax_01_projet_decentralise.webp){: .img-center loading=lazy }

Depuis le mois de mai dernier, un prototype développé au sein de la Fabrique des Géocommuns de l'IGN évolue au gré des tests et des retours des premiers utilisateurs.

> La version bêta actuelle propose les premières briques essentielles que sont l'API et la visionneuse ainsi que deux outils de versement de photos sans oublier le moteur de floutage.

Reviewer: De mémoire, je crois qu'il y a une doc de l'API. Si c'est le cas, je trouverai ça intéressant de le rajouter.

Reviewer: Lien vers la doc de l'API : https://panoramax.ign.fr/api/docs/swagger

Author: top

Tous ces développements sont libres et open-source, publiés sur <https://gitlab.com/geovisio> pour permettre aussi un retour direct avec des issues ou de la documentation.

----

## Une instance Panoramax : stockage et API

Partager des photos c'est avant tout une question de stockage et de catalogage de celles-ci.

![Panoramax - Carte des contributions](https://cdn.geotribu.fr/img/articles-blog-rdp/articles/2023/panoramax/panoramax_02_carte.webp){: .img-center loading=lazy }

> Une API permet de verser des images, de les cataloguer, puis de les rechercher pour une diffusion à destination de différents outils (visionneuse mais pas uniquement). Elle s'appuie sur un espace de stockage et une base de données (PostgreSQL). Cette API se conforme au standard [STAC](https://stacspec.org/) (_Spatio Temporal Assets Catalog_) pour assurer l'intéropérabilité avec des outils existants.

Reviewer 3: Lien vers l'API / sa documentation / canal du forum ??

Author: fait :-)

Toute personne ou entité peut déployer sa propre "instance" Panoramax. Ceci permet par exemple à une collectivité de diffuser ses prises de vues en toute autonomie.

Actuellement deux instances publiques ont été mises en place, l'une par l'IGN et l'autre par OpenStreetMap France. Le versement d'images y est ouvert à tous.

Les instances de stockage telles que celles de l'IGN ou d'OSM permettent à ceux qui n'ont pas la capacité de créer leur propre instance d'y stocker et partager leurs photos.

> En quelques mois, ce sont plus de 8 millions de photos qui ont ainsi été versé sur ces deux instances.

Reviewer 4: accord de versé avec photos ou 8 millions donc versées ou versés

> En quelques mois, ce sont plus de 8 millions de photos qui ont ainsi été versées sur ces deux instances.

Reviewer: ```suggestion
En quelques mois, ce sont plus de 8 millions de photos qui ont ainsi été versées sur ces deux instances.

[Instance IGN :fontawesome-solid-image:](https://panoramax.ign.fr/){: .md-button }
[Instance OpenStreetMap :fontawesome-solid-image:](https://panoramax.openstreetmap.fr){: .md-button }
{: align=middle }
```

Permet d'ajouter des boutons.

----

## Un viewer de référence

Stocker et cataloguer c'est bien, mais permettre la visualisation des photos est bien sûr une nécessité !

> La visionneuse de référence actuelle est issue de Geovisio, un développement initié entre autre par Géovélo et largement amélioré ces derniers mois pour Panoramax.

Reviewer 4: ajout d'un hyperlien vers géovélo

Reviewer 3: J'aurai pas dit mieux

- https://geovisio.fr/
- https://geovelo.app/fr/

Elle permet de naviguer sur un fond de carte, de visualiser les séquences de photos et les photos qu'elles soient classiques ou sphériques (vues sphériques immersives à 360°).

![Panoramax - Visualiseur](https://cdn.geotribu.fr/img/articles-blog-rdp/articles/2023/panoramax/panoramax_03_viewer.webp){: .img-center loading=lazy }

----

## Les outils de versement

> Le premier outil de version proposé par la startup Panoramax a visé les versements en masse de prises de vues existantes. C'est un outil en ligne de commande (écrit en python) qui permet de verser des dossiers entiers d'images.

Reviewer 4: version = versement ?
Le cas échéant, je propose de n'écrire que "le premier outil proposé" pour éviter la répétition de versement dans la phrase

> Le premier outil de version proposé par la startup Panoramax a visé les versements en masse de prises de vues existantes. C'est un outil en ligne de commande (écrit en python) qui permet de verser des dossiers entiers d'images.

Reviewer 3: On "apprend" ici que c'est une startup ... j'imagine qu'il s'agît d'une startup d'état (intrapreneur.e.s ?) Si c'est utile  mettre un lien vers la startup, sinon, on peut aussi en rester au projet de géo-commun ?

Reviewer 3: Ajouter des liens vers les outils de versement ? Premier et second ?

Depuis début septembre, un second outil de versement est disponible sous la forme d'une interface web utilisable avec un simple navigateur. On peut y glisser-déposer les photos à verser, qui sont ensuite envoyées par le navigateur vers l'instance Panoramax.

![Panoramax - Téléversement via l'application web](https://cdn.geotribu.fr/img/articles-blog-rdp/articles/2023/panoramax/panoramax_04_upload.webp){: .img-center loading=lazy }

> Les versements ne sont pas anonymes, un compte utilisateur est nécessaire, pour l'instance OpenStreetMap France, c'est tout simplement le compte OSM qui est utilisé pour s'identifier.

Reviewer 3: Quelle authentification pour l'instance IGN ?

Reviewer: > Quelle authentification pour l'instance IGN ?

Il faut créer un compte dédié mais ça vaut le coup de le préciser.

Author: ok

----

## Une API de floutage

Le floutage des plaques d'immatriculation et des visages est un pré-requis à la diffusion publique des photos de terrain.

> Celui-ci se fait à l'aide d'un modèle de machine-learning qui a été entraîné pour l'occasion, collaborativement. Quelques milliers de photos on été annotées par une vingtaine de contributeurs à l'aide du logiciel libre label-studio qui ont été utilisées pour entraîner un modèle avec YOLOv8.

Reviewer 4: typo : "onT été annotées" et j'accorderais plutôt avec milliers donc annotés

Author: pas joli joli c'est vrai

> Celui-ci se fait à l'aide d'un modèle de machine-learning qui a été entraîné pour l'occasion, collaborativement. Quelques milliers de photos on été annotées par une vingtaine de contributeurs à l'aide du logiciel libre label-studio qui ont été utilisées pour entraîner un modèle avec YOLOv8.

Reviewer 4: Ajout d'un hyperlien vers label-studio ?

> Celui-ci se fait à l'aide d'un modèle de machine-learning qui a été entraîné pour l'occasion, collaborativement. Quelques milliers de photos on été annotées par une vingtaine de contributeurs à l'aide du logiciel libre label-studio qui ont été utilisées pour entraîner un modèle avec YOLOv8.

Reviewer 4: Je serais d'avis de couper la dernière phrase en deux.
Quelques milliers de photos on été annotées par une vingtaine de contributeurs à l'aide du logiciel libre label-studio.
Celles-ci ont été utilisées pour entraîner un modèle avec YOLOv8.

> Celui-ci se fait à l'aide d'un modèle de machine-learning qui a été entraîné collaborativement. Pour cela, quelques milliers de photos ont été annotées par une vingtaine de contributeurs à l'aide du logiciel libre label-studio. Par la suite, les images classifiées ont été utilisées pour entraîner un modèle avec YOLOv8.

Reviewer 3: Ajout des liens vers label-studio et YOLOv8 ?

![Panoramax - API de floutage](https://cdn.geotribu.fr/img/articles-blog-rdp/articles/2023/panoramax/panoramax_05_floutage.webp){: .img-center loading=lazy }

> Le floutage a donné de suite des bons résultats sur les photos "planes" mais était un peu décevant sur les photosphères (360°) en partie à cause de leur dimensions. Un changement de paramètre permet désormais de bien meilleurs résultats et le modèle a été à nouveau entrâiné pour encore améliorer les résultats.

Reviewer 4: - typo : entraîné
- je trouve étrange l'emploi de "permet" je dirais "donne"
- répétition du mot résultat dans la phrase, remplacer le dernier par "taux de détection" ou "qualité de détection" ?

> Le floutage a donné de suite de bons résultats sur les photos "planes" mais était un peu décevant sur les photosphères (360°) en partie à cause de leurs dimensions. Un changement de paramètre permet désormais de bien meilleurs résultats et le modèle a été à nouveau entrainé pour encore améliorer les résultats.

Reviewer 3: Peut être expliquer / illustrer brièvement les défauts corrigés ou le paramètre ?

Author: ce serait bien de donner des détails dans un article complémentaire sur le floutage

> Les annotations et le modèle couvrent un troisième type d'objet: les panneaux de signalisation. Ceci a permis de démarrer une expérimentation pour les détecter et les classer automatiquement.

Reviewer 4: ajouter un espace avant le :

----

## La gestion des séquences

Une fois les séquences de photos versées sur une instance, quelques outils sont disponibles pour masquer ou supprimer si besoin des photos ou des séquences entières.

![Panoramax - Gestion des séquences](https://cdn.geotribu.fr/img/articles-blog-rdp/articles/2023/panoramax/panoramax_06_sequences.webp){: .img-center loading=lazy }

De premiers outils de filtrage ont été mis en place dans la visionneuse pour sélectionner des images par date ou type. Ceci sera complété au fur et à mesure des besoins identifiés.

----

> ## Quelles sont les perspectives de réutilisation?

Reviewer 4: ajouter un espace avant le ?

La contribution est la clé du succès du géocommun. Mais la réussite du projet tient aussi à ses réutilisations, et les possibilités sont infinies. Voici quelques exemples :

* Base de données Panneaux : référencement et identification automatisée des panneaux routiers

> * Usages non routiers : référentiel des rivières (ex. dans le Lot Dordou), chemins de randonnée.

Reviewer 3: ex. dans le _bassin_ Lot-Dordou ou plutôt : ex. dans le [Lot Lozérien](https://lot-dourdou.fr/nos-cours-deau-en-vues-immersives/) mis en place par le [SMLD](https://lot-dourdou.fr/) 

C'est le nom du syndicat mais c'est aussi comme dire dans la Seine-Marne ou dans l'Adour-Garonne.

Author: ok merci

![Panoramax - Usage non routier](https://cdn.geotribu.fr/img/articles-blog-rdp/articles/2023/panoramax/panoramax_07_canal.webp){: .img-center loading=lazy }

> * Enseignes publicitaires : détection par les collectivités locales

Reviewer 3: Ce serait chouette d'avoir les cas d'usage : cela inciterait encore plus à contribuer.

Author: trop tôt pour le moment...

* Boitiers Fibre : repérages avant intervention
* Etat des fossés : repérages pour intervention
* végétation : identifier les tailles nécessaires - état (taille nécessaire)
* cimetières : vérifier l'état des sépultures
* accessibilité : préparer un déplacement, vérifier les cheminements
* arboretum - tourisme : se répérer avant la visite
* arrêts de bus : repérages, état

> 

Reviewer 4: Par cohérence, commencer toutes (ou aucunes) les énums par une majuscule

> Et beaucoup d'autres encore à inventer. À suivre !

Reviewer: ```suggestion
Et beaucoup d'autres encore à inventer. À suivre !

[Forum des Géocommuns - Panoramax :fontawesome-solid-comment:](https://forum.geocommuns.fr/c/panoramax/6){: .md-button }
{: align=middle }
```

Permet d'ajouter un bouton qui renvoie vers le forum.

---
title: "Panoramax, quel bilan tirer de la phase de construction ?"
subtitle: Photo-cartographions librement le monde
authors:
    - Mathilde FERREY
    - Christian QUEST
categories:
    - article
comments: true
date: "2023-10-03 10:20"
description: "C'était il y a un an... l'IGN répondait à la proposition d'OpenStreetMap France et se lançait dans une phase d'investigation pour la création d'un street-view libre. Aujourd'hui, où en est-on ?"
icon: material/panorama-variant

> image: "Image d'illustration de l'article qui sert ensuite dans la mise en avant : réseaux sociaux, flux RSS... 400x800 en PNG"

Reviewer 2: Il manque l'image d'en-tête : https://contribuer.geotribu.fr/guides/metadata_yaml_frontmatter/. Laquelle voulez-vous utiliser ?

A noter que si le champ est laissé vide, une image est générée automatiquement.

> license: default

Reviewer 2: @mferrey @cquest êtes-vous OK avec la licence par défaut ?

https://contribuer.geotribu.fr/guides/licensing/#licence-par-defaut

robots: index, follow
tags:
    - Geovisio
    - IGN
    - OpenStreetMap
    - Panoramax
    - vues immersives
---

> # Panoramax, quel bilan tirer de la phase de construction?

Reviewer 4: Il faudrait ajouter un espace avant le ? ... oui je sais je suis pénible avec ça.

> :calendar: Date de publication initiale : 3 octobre 2023

Reviewer 4: Date de publication à modifier...on a hâte de la voir en ligne !

Les instances de stockage telles que celles de l'IGN ou d'OSM permettent à ceux qui n'ont pas la capacité de créer leur propre instance d'y stocker et partager leurs photos.

En quelques mois, ce sont plus de 8 millions de photos qui ont ainsi été versées sur ces deux instances.

----

## Un viewer de référence

Stocker et cataloguer c'est bien, mais permettre la visualisation des photos est bien sûr une nécessité !

La visionneuse de référence actuelle est issue de [Geovisio](https://geovisio.fr/), un développement initié entre autre par [Géovélo](https://geovelo.app/fr/) et largement amélioré ces derniers mois pour Panoramax.

Elle permet de naviguer sur un fond de carte, de visualiser les séquences de photos et les photos qu'elles soient classiques ou sphériques (vues sphériques immersives à 360°).

![Panoramax - Visualiseur](https://cdn.geotribu.fr/img/articles-blog-rdp/articles/2023/panoramax/panoramax_03_viewer.webp){: .img-center loading=lazy }

----

## Les outils de versement

> Le premier outil proposé par [la start-up d'état de Panoramax](<https://beta.gouv.fr/startups/vues-immersives-libres.html>) a visé les versements en masse de prises de vue existantes. C'est [un outil en ligne de commande](<https://gitlab.com/geovisio/cli>) (écrit en python) qui permet de verser des dossiers entiers d'images.

Reviewer 4: start-up d'État

La contribution est la clé du succès du géocommun. Mais la réussite du projet tient aussi à ses réutilisations, et les possibilités sont infinies. Voici quelques exemples :

* Base de données Panneaux : référencement et identification automatisée des panneaux routiers
* Usages non routiers : référentiel des rivières (ex. dans le [Lot Lozérien](https://lot-dourdou.fr/nos-cours-deau-en-vues-immersives/) mis en place par le [SMLD](https://lot-dourdou.fr/le-syndicat/)), chemins de randonnée.

![Panoramax - Usage non routier](https://cdn.geotribu.fr/img/articles-blog-rdp/articles/2023/panoramax/panoramax_07_canal.webp){: .img-center loading=lazy }

* Enseignes publicitaires : détection par les collectivités locales
* Boitiers Fibre : repérages avant intervention
* Etat des fossés : repérages pour intervention
* Végétation : identifier les tailles nécessaires - état (taille nécessaire)
* Cimetières : vérifier l'état des sépultures
* Accessibilité : préparer un déplacement, vérifier les cheminements
* Arboretum - tourisme : se répérer avant la visite
* Arrêts de bus : repérages, état

Et beaucoup d'autres encore à inventer. À suivre !

----

> ## Autrice
> 
> --8<-- "content/team/mfer.md"

Reviewer 2: ```suggestion
## Auteurs

--8<-- "content/team/mfer.md"  
--8<-- "content/team/cque.md"
```
